lib/src/object_id.rs

```diff
@@ -103,14 +103,19 @@ macro_rules! impl_id_type {
 
 pub(crate) use {id_type, impl_id_type};
 
+/// An identifier prefix (typically from a type implementing the [`ObjectId`]
+/// trait) with facilities for converting between bytes and a hex string.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct HexPrefix {
-    // For odd-length prefix, lower 4 bits of the last byte is padded with 0
+    // For odd-length prefixes, the lower 4 bits of the last byte are
+    // zero-filled (e.g. the prefix "abc" is stored in two bytes as 0xabc0).
```

Comment:
nit: `b"\xab\xc0" or simply `"abc0"` to avoid endianness stuff?

Reply:
Thanks, good point. I'll change it to "abc".

Reply:
Sorry, typo. Changing to "abc0".